.pre-commit-config.yaml

```diff
@@ -11,20 +11,20 @@ repos:
       - id: end-of-file-fixer
 
   - repo: https://github.com/omnilib/ufmt
-    rev: v1.3.3
+    rev: v2.1.0
     hooks:
       - id: ufmt
         additional_dependencies:
-          - black == 22.3.0
-          - usort == 1.0.2
+          - black == 23.3.0
+          - usort == 1.0.7
```

Comment:
We can't, unfortunately.
We have to stick to what is used by `pyfmt` internally

```
pyf%                                                                                                                   nicolashug@devvm779 ~ ❯❯❯ pyfmt --version
fb-pyfmt built mode/opt (xar) on Python 3.10.9+fb (platform010)
ufmt version 2.0.1
black version 22.12.0
usort version 1.0.5
libcst version 0.4.10
```

Reply:
Updated the config to match the `pyfmt` requirements.